doc/source/release/2.0.0-notes.rst

```diff
@@ -395,7 +395,7 @@ replaced with ``NPY_RAVEL_AXIS``. See also :ref:`migration_maxdims`.
 ``NPY_MAXARGS`` not constant and ``PyArrayMultiIterObject`` size change
 -----------------------------------------------------------------------
 Since ``NPY_MAXARGS`` was increased, it is now a runtime constant and not
-compiletime constant anymore.
+compile time constant anymore.
```

Comment:
If we're going to change this, consider hyphenating (as a compound modifier) for clarity.
```suggestion
compile-time constant anymore.
```

Reply:
done